quizizz.py

```
@@ -0,0 +1,41 @@
+import requests
+import random
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)